Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Transformer.purs

```diff
@@ -1049,6 +1050,6 @@ getFareProductType fareProductType =
 
 getServiceNames :: Array EstimateAPIEntity -> String -> Array String
 getServiceNames estimate  prefervariant =
-  let filter = if prefervariant == "AUTO_RICKSHAW" then  ["AUTO_RICKSHAW" , "TAXI"] else ["SEDAN", "TAXI", "TAXI_PLUS"]
-      array  =  foldl (\acc (EstimateAPIEntity  estimate) -> if  DA.elem estimate.vehicleVariant filter then acc <> [fromMaybe "" estimate.serviceTierName] else acc ) [] estimate 
+  let filter = if prefervariant == "Auto" then  ["Auto" , "Non-AC Mini"] else ["Sedan", "Non-AC Mini", "AC Mini"]
```

Comment:
Can we rename `prefervariant`, this suggests Vehicle Variant Name whereas we are doing a check on service tier name?